modules/installation-complete-user-infra.adoc

```diff
@@ -146,19 +146,23 @@ command.
 If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
 
-ifndef::ibm-power[]
-. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing. Do not enable multipathing during installation.
+ifndef::ibm-power,ibm-z[]
+. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing.
 +
 [NOTE]
 ====
 When installing with multipath, it is strongly recommended to enable it at installation time, and not at a later time, which can cause problems.
 ====
 +
 See "Enabling multipathing with kernel arguments on {op-system}" in the _Installing on bare metal_ documentation for more information.
-endif::ibm-power[]
+endif::ibm-power,ibm-z[]
```

Comment:
s/endif::ibm-power,ibm-z[]/endif::ibm-power,ibm-z,ibm-z-kvm[]

At the bottom of the file where I can't comment again add: 

```
ifeval::["{context}" == "installing-ibm-z-kvm"]
:!ibm-z-kvm:
endif::[]
ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
:!ibm-z-kvm:
:!restricted:
endif::[]
```